checker/src/org/checkerframework/checker/fenum/FenumAnnotatedTypeFactory.java

```diff
@@ -50,7 +50,11 @@ public FenumAnnotatedTypeFactory(BaseTypeChecker checker) {
         // load individually named qualifiers
         if (qualNames != null) {
             for (String qualName : qualNames.split(",")) {
-                qualSet.add(loader.loadExternalAnnotationClass(qualName));
+                Class<? extends Annotation> loadedClass =
```

Comment:
Undo null checks, because its never null, internally caught in loadExternalAnnotationClass.